app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :body, presence: true, length: { maximum: 300 }
+  validates :user, presence: true
+  validates :post, presence: true
```

Comment:
- [ ] these validations are unnecessary because belongs_to already validates the presence